### PR TITLE
Finalized template

### PR DESCRIPTION
**Pull request by eimfach**
PR for overview , please do a quick review

### CODE REVIEW

File: bin/compile.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const fs = require('fs-extra');
 const paths = config.utils_paths;
 
 debug('Create webpack compiler.');
-const compiler = require('webpack')(require('../build/webpack.config'));
+const compiler = require('webpack')(require('../config/webpack.config.js'));
```
Comment (layflags) [inline bin/compile.js]: do we need `.js` in require param?

File: package.json [hunk 2]
```diff
@@ -52,93 +52,93 @@
...
+    "sinon": "1.17.2",
+    "sinon-chai": "2.8.0",
+    "style-loader": "0.13.0",
+    "url-loader": "0.5.6",
     "webpack": "1.12.9",
-    "webpack-dev-middleware": "^1.4.0",
-    "webpack-hot-middleware": "^2.6.0"
+    "webpack-dev-middleware": "1.4.0",
+    "webpack-hot-middleware": "2.6.0"
   }
```
Comment (layflags) [inline package.json]: you fixed the version of every dependency ... why that? Do you fear non-semver compliance?

File: src/main.js [hunk 3]
```diff
@@ -12,10 +12,10 @@ const initialState = window.__INITIAL_STATE__;
 
 export default {
   /**
-   * (Sample) Create this widget & append it to the DOM
+   * Initializes & Renders the MerchantApp
```
Comment (layflags) [inline src/main.js]: ~~MerchantApp~~ -> App / Widget

**Comment by layflags** [comment]
nice dir structure now!

File: package.json (hunk 2 above)
Comment (eimfach) [inline package.json]: Yes, it's not only that I fear it, it's just that we had actually some crazy breaking errors in some modules, there are also issues on the origin repo for that. I just talked about Maruf with that and we will just make the template in it's dependecies static. As you work with it in another project, you the can manage deps on your own. It's just that the template comes 'plug and play'.

File: bin/compile.js (hunk 1 above)
Comment (eimfach) [inline bin/compile.js]: right

Comment (eimfach) [inline bin/compile.js]: fixed

File: src/main.js (hunk 3 above)
Comment (eimfach) [inline src/main.js]: fixed

**Pull request merged**